### PR TITLE
Mobile menu 

### PR DESCRIPTION
**Pull request by hydaline-code**
🙌 Hello hope everything is okay, in this part of my work, you will find the hamburger menu done with javascript for its interactivity. The corresponding linter checks for CSS and javascript is okay kindly appreciate by giving it a review for improvement .thanks

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  close1.style.display = 'none';
+  newmenu.style.display = 'none';
+});
+
+for (let i = 0; i < item.length; i += 1) {
+  item[i].addEventListener('click', () => {
+    newmenu.style.display = 'none';
+    view1.style.display = 'block';
+  });
+}
```
Comment (GzimAsani) [inline index.js]: - [ ] When I click one of the items on the menu I get redirected correctly, but then the X is present and not the hamburger menu. Kindly fix this 
https://www.awesomescreenshot.com/video/15393281?key=9d6fa7e058bdccd9568869407f132248

Comment (GzimAsani) [reply index.js]: - [ ] The menu should extend all the way down and when opened you should be able to see only the menu, also please work on design to make it more similar 

Yours:

image

Design:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @hydaline-code                    :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your styling, it is almost the same  :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @hydaline-code , 
image

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**